std/variant.d

```diff
@@ -2204,6 +2247,13 @@ private auto visitImpl(bool Strict, VariantType, Handler...)(VariantType variant
                         result.indices[tidx] = dgidx;
                     }
                 }
+                else static if (isSomeFunction!(dg!T))
+                {
+                    assert(result.generalFuncIdx == -1 ||
+                           result.generalFuncIdx == dgidx,
+                           "Provide no more than 1 generic (templated) visitor");
```

Comment:
This error message seems poor to me. What about something like "Only one generic visitor function is allowed"?